pyramid_orb/utils.py

```diff
@@ -88,7 +88,8 @@ def get_context(request, model=None):
 
         # extract payload items
         for key, value in param_values.items():
-            schema_object = model.schema().column(key, raise_=False) or model.schema().collector(key)
```

Comment:
the way this works is that if I go to a route, like `/api/v1/<model>`, it will try to map any arguments that are provided to schema specific values.  So if the model has a `name` attribute, then it would know that if I provide `name=XYZ` that it should map to that parameter on the model itself.  It is a way of generating queries for selects, or creation / update values.  any additional parameters would be processed through context (where, limit, locale, etc.).  beyond that, the parameters are dropped and not passed to the ORB system.  If you want to access generic params, you would have to do that by referencing the request directly which is stored in the context's scope